Resources/Prototypes/Entities/Objects/Weapons/Guns/SMGs/smgs.yml

```
@@ -62,7 +62,7 @@
   name: Atreides
   parent: [BaseWeaponSubMachineGun, BaseMajorContraband]
   id: WeaponSubMachineGunAtreides
-  description: Pla-ket-ket-ket-ket! Uses .35 auto ammo.
+  description: A discontinued and very rare ‘machine pistol’ dating back to The Corporate War. It's extremely high fire rate and compact profile make it useful in close quarters combat. Despite its age, it is in remarkably good condition. Feeds from .35 SMG magazines.
```

Machine pistol is a common term used in firearms, no reason to put it in quotes, but you can keep it if it looks cool.

The Corporate Wars is commonly referred to in plural as it involved multiple corporations.

```suggestion
  description: A discontinued and very rare ‘machine pistol’ dating back to the Corporate Wars. It's extremely high fire rate and compact profile make it useful in close quarters combat. Despite its age, it is in remarkably good condition. Feeds from .35 SMG magazines.
```